internal/scripts/execute.go

```diff
@@ -100,3 +98,12 @@ func execute(
 
 	return &scriptResult{value}, nil
 }
+
+func executeLocalScript(args []string, filename string, readerWriter flowkit.ReaderWriter, flow flowkit.Services) (command.Result, error) {
```

Comment:
do you need this function?

Reply:
Good catch. I had these separated before it was moved to the new commands and needs to be removed now.